Sources/_InternalTestSupport/SwiftPMProduct.swift

```diff
@@ -82,7 +82,8 @@ extension SwiftPM {
     public func execute(
         _ args: [String] = [],
         packagePath: AbsolutePath? = nil,
-        env: Environment? = nil
+        env: Environment? = nil,
+        errorIfCommandUnsuccessful: Bool = true
```

Comment:
I'm sure there's a more elegant way to do this, no?

Reply:
Likely. Can you offer a suggestion?